packages/manager/apps/telecom/src/components/sidebar/sidebar.config.js

```diff
@@ -205,14 +205,8 @@ angular.module('managerApp').config((SidebarMenuProvider) => {
         external: true,
         onClick: setTracker('order-O365_Sharepoint', 'navigation', 'Telecom', 'telecom'),
       }],
-    }, {
-      title: $translate.instant('telecom_sidebar_actions_menu_hubic'),
```

Comment:
We can also remove the unused translation from `packages/manager/apps/telecom/src/components/sidebar/translations/Messages_fr_FR.json` file.